platform/web/display_server_web.cpp

```diff
@@ -902,8 +902,10 @@ void DisplayServerWeb::process_joypads() {
 		for (int b = 0; b < s_btns_num; b++) {
 			// Buttons 6 and 7 in the standard mapping need to be
 			// axis to be handled as JoyAxis::TRIGGER by Godot.
-			if (s_standard && (b == 6 || b == 7)) {
-				input->joy_axis(idx, (JoyAxis)b, s_btns[b]);
+			if (s_standard && (b == 6)) {
+				input->joy_axis(idx, JoyAxis::TRIGGER_LEFT, s_btns[b]);
+			} else if (s_standard && (b == 7)) {
+				input->joy_axis(idx, JoyAxis::TRIGGER_RIGHT, s_btns[b]);
```

Comment:
Doesn't the change to `godotcontrollerdb.txt` make this unused?
It should not get `b6` and `b7` anymore.

Reply:
It's the other way around. Lefttrigger:+a4 means that when we receive +a4 from the browser we map it to lefttrigger in godot.
But the browser will never send that, it sends b6. That's why we need to convert it.

The "proper" solution is not lying in de controllerdb, not converting any buttons to axes, and modifying input->joy_button so that it can properly handle analog buttons. 

I'll give it a try later.

Reply:
Ok reading the code a bit more, the proper fix basically means implementing analog button support in Godot

The following need to be changed:

`Input::joy_button` Needs to take a float p_value instead of a bool p_pressed, plus implementation changes
`Input::Joypad` last_buttons[] needs to be array of floats instead of bools
`Input::JoyEvent Input::_get_mapped_button_event` needs to handle converting analog buttons to full and half axes
`Input::_button_event` needs to use set_pressure() instead of only set_pressed()
At least `InputEventJoypadButton` seems to actually support analog values through it's `pressure` property for some reason. 

I think it's too big a change for a quirk of the html gamepad api and it'll probably break something else, so I'd just keep this PR as is, which essentially does what the original code meant to do.